### ISSUE TITLE
Package license file

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/epics-striptool-feedstock#1]
As noted in the `staged-recipes` example, [packages should contain a license file]( https://github.com/conda-forge/staged-recipes/blob/12f27ec6030558f350f8eeaae7a24094052cb3e1/recipes/example/meta.yaml#L81-L85 ). Currently this feedstock [does not list one]( https://github.com/conda-forge/epics-striptool-feedstock/blob/1a0875ec66109a2415b163dadcaa9512097927d3/recipe/meta.yaml#L43-L46 ). Please update to include a license file.

**Comment by pthomasj** [comment]
The maintainers of the code added a license file to the repo, but did not create a new version or tarball for it, so I copied the license file and added it to the recipe.

**Comment by jakirkham** [comment]
Yep that makes sense. Whenever they make a new release, the vendored copy can be dropped and the one contained in the source code can be used
